contracts/multihop/src/tests/swap.rs

```diff
@@ -13,27 +13,11 @@ fn swap_three_equal_pools_no_fees() {
 
     let admin = Address::random(&env);
 
-    let mut token1 = deploy_token_contract(&env, &admin);
-    let mut token2 = deploy_token_contract(&env, &admin);
-    let mut token3 = deploy_token_contract(&env, &admin);
-    let mut token4 = deploy_token_contract(&env, &admin);
-
-    env.mock_all_auths();
-    env.budget().reset_unlimited();
-
-    let mut tokens = [&mut token1, &mut token2, &mut token3, &mut token4];
-    tokens.sort_by(|a, b| a.address.cmp(&b.address));
```

Comment:
Actually it doesn't make sense to sort those tokens here, it needs to be done just in a helper of creating liquidity pool. That's where it matters.

Reply:
Done.